stock_mech_files/GRIMech30.yaml

```diff
@@ -0,0 +1,1817 @@
+description: |-
```

Comment:
I do not believe we should commit these.  They can be/will be specific for each model (such as modifications of species order or rates).